pkgs/development/compilers/llvm/17/llvm/default.nix

```diff
@@ -313,6 +314,7 @@ stdenv.mkDerivation (rec {
     "-DLLVM_HOST_TRIPLE=${stdenv.hostPlatform.config}"
     "-DLLVM_DEFAULT_TARGET_TRIPLE=${stdenv.hostPlatform.config}"
     "-DLLVM_ENABLE_DUMP=ON"
+    "-DLLVM_ENABLE_LIBEDIT=FORCE_ON"
```

Comment:
Is FORCE_ON necessary? If it is, can you conditionalize it on `libedit != null` so that it can be switched off with `.override { libedit = null; }`.